src/commands/reminders/cancelreminder.ts

```diff
@@ -1,34 +1,44 @@
 import { Reminder } from '@lib/types/Reminder';
 import { DB } from '@root/config';
-import { Message } from 'discord.js';
+import { ApplicationCommandOptionData, CommandInteraction, Message } from 'discord.js';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
 	description = 'Cancel any pending reminders you may have.';
 	usage = '<reminder number>';
 	extendedHelp = 'You can only cancel one reminder at a time';
-	aliases = ['cr', 'removereminder'];
 
-	run(msg: Message, [reminder]: [Reminder]): Promise<Message> {
-		msg.client.mongo.collection(DB.REMINDERS).findOneAndDelete(reminder);
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'remindernumber',
+			type: 'INTEGER',
+			required: true,
+			description: 'Index of the reminder to cancel'
```

Comment:
```suggestion
			description: 'ID of the reminder to cancel'
```
Just a slight change for improved clarity. I can't lay my finger on it precisely, but `index` sounds kind of vague/misleading.